packages/@o3r/localization/schematics/ng-update/v10-0/index.ts

```diff
@@ -36,4 +36,5 @@ function updateV10_0Fn(): Rule {
 /**
  * Update of Otter library V10.0
  */
+// eslint-disable-next-line @typescript-eslint/naming-convention -- contains the version number
```

Comment:
is it not better to have it at the top of the file ?
We might still add migration functions.

Reply:
That's something we discussed with the team and we said that it's better to disable only where it's needed
If you want we can re-discussed it to set a global config on the repo to disable this rule for all files inside an `schematics/ng-update` folder